test/integration/migration.spec.js

```diff
@@ -171,19 +172,60 @@ describe('the migration', function () {
     ]);
   }));
 
+  it('changes field IDs', co(function * () {
+    yield migrator(idChange);
+
+    const resultAfterModify = yield request({
+      method: 'GET',
+      url: '/content_types/dog',
+      headers: {
+        'X-Contentful-Beta-Dev-Spaces': 1
+      }
+    });
+
+    expect(resultAfterModify.fields).to.eql([
+      {
+        id: 'goodboys',
```

Comment:
Shouldn't this be `aDifferentId`?

Reply:
No, because in this example I switched the id back after changing the name. But it is a bit contrived, I'll change the test to the more normal path.

Reply:
Ah I missed the last line where it's renamed back (actually that makes a point for not allowing to rename the same field twice in a "circular" manner)